rsocket/internal/Common.h

```diff
@@ -124,6 +124,7 @@ struct ProtocolVersion {
 
   static const ProtocolVersion Unknown;
   static const ProtocolVersion Latest;
+  static const ProtocolVersion Current();
```

Comment:
Like where this function resides now.  It did not make sense to have it in FrameSerializer.